testnets/airchainstestnet/chain.json

```diff
@@ -0,0 +1,368 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "airchainstestnet",
+  "status": "live",
+  "website": "https://airchains.io/",
+  "network_type": "testnet",
+  "pretty_name": "Airchains Switchyard Testnet",
+  "chain_id": "junction",
+  "bech32_prefix": "air",
+  "daemon_name": "junctiond",
+  "node_home": "$HOME/.junctiond",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "amf",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.005,
+        "high_gas_price": 0.01
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "amf"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/airchains-network/junction",
+    "recommended_version": "v0.1.0",
+    "cosmos_sdk_version": "0.50.3",
+    "genesis": {
+      "genesis_url": "https://github.com/airchains-network/junction/releases/download/v0.1.0/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v1",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ]
+      }
+    ]
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/airchainstestnet/images/airchains.png",
```

Comment:
there is no file of this name. I assume this should include the "-logo"...

Reply:
Yes, you are absolutely correct. I forgot to include that information. I have now updated it. Please review it and let me know if it is accurate.